features/networking/egress-ip.feature

```diff
@@ -66,26 +66,65 @@ Feature: Egress IP related features
     Then the step should succeed
     Given 4 pods become ready with labels:
       | name=test-pods |
-    And evaluation of `pod(0).name` is stored in the :pod1 clipboard
-    And evaluation of `pod(1).name` is stored in the :pod2 clipboard
     And evaluation of `pod(2).name` is stored in the :pod3 clipboard
     And evaluation of `pod(3).name` is stored in the :pod4 clipboard
 
     # try to access the receiver service to get the source IP
+
     When I execute on the "<%= cb.pod1 %>" pod:
-      | curl | -s | --connect-timeout | 5 | <%= cb.ipecho_ip %>:8888 |
+      | curl | -s | --connect-timeout | 5 | ifconfig.me |
     Then the step should succeed
     And the output should contain "<%= cb.valid_ip %>"
     When I execute on the "<%= cb.pod2 %>" pod:
-      | curl | -s | --connect-timeout | 5 | <%= cb.ipecho_ip %>:8888 |
+      | curl | -s | --connect-timeout | 5 | ifconfig.me |
     Then the step should succeed
     And the output should contain "<%= cb.valid_ip %>"
     When I execute on the "<%= cb.pod3 %>" pod:
-      | curl | -s | --connect-timeout | 5 | <%= cb.ipecho_ip %>:8888 |
+      | curl | -s | --connect-timeout | 5 | ifconfig.me |
     Then the step should succeed
     And the output should contain "<%= cb.valid_ip %>"
     When I execute on the "<%= cb.pod4 %>" pod:
-      | curl | -s | --connect-timeout | 5 | <%= cb.ipecho_ip %>:8888 |
+      | curl | -s | --connect-timeout | 5 | ifconfig.me |
     Then the step should succeed
     And the output should contain "<%= cb.valid_ip %>"
 
+  # @author huirwang@redhat.com
+  # @case_id OCP-15472
+  @admin
+  Scenario: The egressIPs will be added to the node's primary NIC when it gets set on hostsubnet and will be removed after gets unset
+    # add the egress ip to the hostsubnet
+    Given  the valid egress IP is added to the node
+    And evaluation of `node.name` is stored in the :egress_node clipboard
+
+    # add the egress ip to the project
+    Given I have a project
+    And evaluation of `project.name` is stored in the :project clipboard
+    When I run the :patch admin command with:
+      | resource      | netnamespace                            |
+      | resource_name | <%= cb.project %>                       |
+      | p             | {"egressIPs":["<%= cb.valid_ip %>"] }   |
+      | type          | merge                                   |
+    Then the step should succeed
+
+    # check egress ip was added to primary interface
+    When I run command on the "<%= cb.egress_node %>" node's sdn pod:
+      | bash | -c | ip address show <%= cb.interface %> |
+    Then the step should succeed
+    And the output should contain "<%= cb.valid_ip %>"
+
+    # Remove the egress ip from hostsbunet
+    When I run the :patch admin command with:
```

Comment:
Thanks for automating @huiran0826 , i believe you can replace whole 116-121 into 2 lines like (for example) 

Given as admin I successfully merge patch resource "netnamespace/<%= project.name %>" with: 
    | {"egressIPs": "null"}} |

and similarly shorten 102-107 in two lines

Reply:
Thanks for the comments, updated it.